indico/modules/events/registration/forms.py

```diff
@@ -473,3 +477,60 @@ def validate_remove(self, field):
 
     def is_submitted(self):
         return super().is_submitted() and 'submitted' in request.form
+
+
+class ChangeRegistrationVisibilityForm(IndicoForm):
```

Comment:
I haven't tested it on my instance yet, but this seems like a form to bulk-manage the consent? I'm not sure whether a bulk change for this is a good idea, especially if the event manager has no way to revert hiding someone, and in particular the manager should not be able to give consent on behalf of the user..

Reply:
Hmm I see the problem. This form allows downgrading consent only, and was meant to fulfil one of the original requirements:
> - option for organizer to unpublish individual participant(s) regardless of consent (ie another column for this)
Now I see that this should be an additional attribute of the registration.. I will work on that

Reply:
Yes exactly. But in general I think it may be enough to make this an action when viewing a specific registration. I don't see that much value in doing this in bulk (but no harm in it either)